cmd/sentry.go

```diff
@@ -100,9 +100,10 @@ func initSentry() {
 			ID: userId.String(),
 		})
 		scope.SetTags(map[string]string{
-			"terminal": strconv.FormatBool(ui.IsTerminal()),
-			"ci":       strconv.FormatBool(analytics.IsCI()),
-			"faas":     strconv.FormatBool(analytics.IsFaaS()),
+			"cookie_id": userId.String(),
```

Comment:
This would lead to high cardinality which is not good for tags.

Reply:
Maybe ill add it to extras then.